server/odc-service/src/main/java/com/oceanbase/odc/service/db/session/DefaultDBSessionManage.java

```diff
@@ -300,38 +301,18 @@ private boolean isGlobalClientSessionEnabled(ConnectionSession connectionSession
                 || VersionUtils.isLessThan(obVersion, GLOBAL_CLIENT_SESSION_OB_VERSION_NUMBER)) {
             return false;
         }
-        try {
-            Integer proxyId = getOBProxyConfig(connectionSession, "proxy_id");
-            Integer clientSessionIdVersion = getOBProxyConfig(connectionSession, "client_session_id_version");
-
-            return proxyId != null
-                    && proxyId >= GLOBAL_CLIENT_SESSION_PROXY_ID_MIN
-                    && proxyId <= GLOBAL_CLIENT_SESSION_PROXY_ID_MAX
-                    && clientSessionIdVersion != null
-                    && clientSessionIdVersion == GLOBAL_CLIENT_SESSION_ID_VERSION;
-        } catch (Exception e) {
-            log.warn("Failed to determine if global client session is enabled: {}", e.getMessage());
-            return false;
-        }
-    }
-
-    /**
-     * Gets the value of OBProxy's configuration variable If an exception occurs or the version does not
-     * support, return null.
-     * 
-     * @param connectionSession
-     * @param configName
-     * @return
-     */
-    private Integer getOBProxyConfig(ConnectionSession connectionSession, String configName) {
-        try {
-            return connectionSession.getSyncJdbcExecutor(ConnectionSessionConstants.BACKEND_DS_KEY)
-                    .query("show proxyconfig like '" + configName + "';",
-                            rs -> rs.next() ? rs.getInt("value") : null);
-        } catch (Exception e) {
-            log.warn("Failed to obtain the value of OBProxy's configuration variable: {}", e.getMessage());
-            return null;
-        }
+        // Check whether the global session is open
+        // If the global session is open, the "time" column will be displayed in the result set after
+        // executing the sql statement of "show processlist"
+        return connectionSession.getSyncJdbcExecutor(ConnectionSessionConstants.BACKEND_DS_KEY)
+                .query("show processlist", rs -> {
+                    try {
+                        int columnIndex = rs.findColumn("time");
+                        return columnIndex > 0;
+                    } catch (SQLException e) {
+                        return false;
```

Comment:
Is it necessary to add a log to print the cause of the exception in this code?

Reply:
ok